src/main/java/org/jenkins/plugins/lockableresources/queue/QueuedContextStruct.java

```diff
@@ -38,13 +38,19 @@
 	 */
 	private String resourceDescription;
 
+	/*
+	 * Name of the variable to save the locks taken.
+	 */
+	private String variableName;
```

Comment:
Isn't that already saved as `LockableResourcesStruct.requiredVar`?

Reply:
I can not actually see that LockableResourcesStruct.requiredVar is used anywhere so it is hard to know what the intent is with that variable.

Reply:
@TobiX How do we move on with this?
[JENKINS-50176](https://issues.jenkins-ci.org/browse/JENKINS-50176) is now the highest voted JIRA ticket for ` lockable-resources-plugin` and should be top priority.

Reply:
Well, I now have a better understanding of the code. Since `requiredVar` seems to be a remnant of a feature that doesn't have any configuration frontend anymore, I'll drop this discussion for now - will be cleaned up later.